### ISSUE TITLE
DWQ - UIPA Mapping changes to soil permeability

### ISSUE THREAD

**Issue by rkelson** [agrc/planning-queue#278]
### Benefit

Provide by year, from 2019 to 2024, GIS layers denoting Pervious (undisturbed), Semi-Impervious (disturbed), and Impervious (developed) land within the Utah Inland Port Authority boundary. This will provide information The Division of Water Quality will utilize along side water quality testing in the UIP for their report on the changes to water quality as the area develops.

### Acceptance Criteria

- [ ] DWQ has reviewed and accepted the accuracy of the digitized areas and assigned categories.

### Notes

This task poses problems with utilizing automation, such as AI and feature extraction algorithms, considering this is change over time using different vintages of aerial photography. Additionally, each subsequent year's polygons are based off the previous year.

As discussed with DWQ the manual digitizing will be done at 1:10,000 scale, with more detail in their 4 testing areas.

### Risks

There is always to possibility overtime for scope creep on the project and the evolution of expectations by DWQ once they determine how they are going to report their finding.

### Issue Reference

refs #25

**Comment by rkelson** [comment]
- [ ] Provide DWQ 2019 baseline layer from review
- [ ] Obtain feedback from DWQ on 2019 layer
- [ ] DWQ accepts the 2019 baseline layer as accurate and complete

**Comment by gregbunce** [comment]
2/27/2020 obtained aerial photography for 2022 from Erik Neemann for SLCo

**Comment by nathankota** [comment]
### FY24 Q3 Sprint 6 NOTES
- 2019 imagery digitization is 95% complete

**Comment by nathankota** [comment]
### FY24 Q4 Sprint 3
- Complete digitization

**Comment by rkelson** [comment]
Received feedback on the 2019 layers. I went through all the comments and made all the suggested changes with a few exceptions. There were a couple areas with small strips of vegetation around a building or a corner of a lot that I did not digitize as the areas were relatively small especially at 1:10,000 scale. 

5/28/24 - Asked DWQ if it is okay to proceed to subsequent years and waiting for their reply.

**Comment by rkelson** [comment]
Going to start on 2020 layer copied from 2019 layer

**Comment by rkelson** [comment]
5/30 - Provided a pdf of 2020 work for DWQ to review

Considering they don't provide a lot of feedback on the work being done, I plan to start work on the 2021 layer. I am lucky that 2021 is the last year we did the POC work in the 3 sample areas because the sample area polygons need to be merged into the UIP-wide 2020 layer.

**Comment by rkelson** [comment]
finished '21 layer and moving on to '22 layer. Not sure when the '23 imagery (which is a combination of years similar to '22) will become available. When I finish '22 I will send DWQ PDFs of '21 & '21 to review.

**Comment by rkelson** [comment]
Let DWQ know I've worked through '21 and '22 and the PDF files can be found in the drive folder. Requested that they take a look at '20, '21 & '22 and let us know if we need to make any changes or if they have any questions. This is as far as I can go until we get the '23 imagery from Erik, likely available mid July. 

Below is a snapshot of the category percentages as of '22:
![Image](https://github.com/agrc/planning-queue/assets/6200941/9d3b78ef-00b5-4461-a71d-1a87d48a3ac5)

**Comment by rkelson** [comment]
Received an email from Erik that the Eagleview Fall 2023/Spring 2024 imagery will be available soon

**Comment by rkelson** [comment]
Obtained '23 imagery from Erik

**Comment by steveoh** [comment]
waiting on 2024 imagery

**Comment by rkelson** [comment]
still waiting on 2024 imagery. It is anticipated to be available next month (Dec.)

**Comment by rkelson** [comment]
still waiting on 2024 imagery. It is still anticipated to be available next month (Dec.)

**Comment by rkelson** [comment]
the new 2024 imagery is available. I plan to work on this soon and when I have time.

**Comment by rkelson** [comment]
2024 imagery is ready for use in this project and I've already flagged some areas for editing in the layer. I need to make the time to restart this project including opening dialog with James Harris that I plan to start working on the final year layer and plan to call the project finished once my work is complete.

**Comment by rkelson** [comment]
4/7/2025

- There are 36 hrs left on this project.
- Sent Jim Harris an email about getting ready to digitize the 2024 layer and asked if there is going to be anything else he will need to close out the project once the digitizing is complete and previously agreed upon deliverables are provided. Got an auto-reply that he is out of the office until April 14th. I am going to hold off on working on the project until after he gets back.

**Comment by rkelson** [comment]
I've been working on the final '24 layer. I still need to create the final statistics, redo the PDF graphics by year, and get the project deliverable to DWQ.

**Comment by rkelson** [comment]
5/21 Project deliverables are ready to hand off to DWQ. I requested from them when would be a good time to give them a thumb drive and to show them what I have. Waiting to hear back from Jim or Ben.

**Comment by rkelson** [comment]
I have a meeting with DWQ 5/28 to show them the work that was done and give them a thumb drive of the imagery used for the project and the digitizing work.

**Comment by rkelson** [comment]
5/28/2025 Met with James Harris and Mark Stanger. I gave them a thumb drive with all the imagery I used for the project, the finalized digitized layers for the project from 2019-2024, the project documentation for the digitizing mythology I used, imagery metadata, and a Pro project to view the work that was done.

I am considering this project done however I will leave this open for another couple week. I anticipate Mark Stanger, who will be taking over the project where I left off, will have some questions.

**Comment by rkelson** [comment]
6/9/2025 Had a virtual meeting with Mark. He cannot upgrade to a more recent version of Pro so the Pro project I gave him needed to be rebuilt at his current version. He should be off to the races but I'm keeping this open for one more sprint incase I hear anything more from DWQ on the project.

**Comment by steveoh** [comment]
@chriswnek FYI this extra work is related to the uic attribute rule migration situation requiring them to stay on pro 2.x

**Comment by rkelson** [comment]
I have not heard anything more from DWQ on the project so I am going to consider it finished.
